app/helpers/organization_helper.rb

```diff
@@ -4,4 +4,9 @@ def display_logo_or_name(organization = nil)
     organization ||= current_organization
     organization.logo.attached? ? image_tag(organization.logo, alt: "#{organization.name} logo", class: "organization-logo", style: "max-height:188px") : organization.name
   end
+
+  def display_partner_fields_value(stored_value)
+    display_value, _ = Organization::ALL_PARTIALS.detect { |arr| arr[1] == stored_value }
+    display_value
```

Comment:
This can be written more naturally:

`Organization::ALL_PARTIALS.to_h.key(stored_value)`